packages/web-app/src/components/common/Maps/common/Markers/Components/EntrancePopup.jsx

```diff
@@ -25,17 +28,17 @@ export const EntrancePopup = ({ entrance }) => (
       value={makeCoordinatesValue(entrance.latitude, entrance.longitude)}
       icon={<GpsFixed color="primary" />}
     />
-    {entrance.cave && entrance.cave.depth && (
+    {entrance.depth && (
       <Property
         secondary
-        value={`${entrance.cave.depth} m`}
+        value={`${entrance.depth} m`}
         icon={<CustomIcon size={25} type="depth" />}
       />
     )}
-    {entrance.cave && entrance.cave.length && (
+    {entrance.length && (
       <Property
         secondary
-        value={`${entrance.cave.length} m`}
+        value={`${entrance.length} m`}
         icon={<CustomIcon size={25} type="length" />}
       />
     )}
```

Comment:
Maybe I missed something, but I couldn't test those changes

This marker seems to be used on the main map and on the massif map:
- On the massif map they are not displayed because the API will never return any entrances (the code is commented) [grottocenter-api/api/controllers/v1/massif/find.js](https://github.com/GrottoCenter/grottocenter-api/blob/develop/api/controllers/v1/massif/find.js#L33)
- On the main map the API will also never return the length, depth or cave name because of missing quotes in the SQL Query. Without quotes, select alias are lowercased (idCave -> idcave) but the code doesn't expect the case to have changed ([grottocenter-api/api/services/GeoLocService.js](https://github.com/GrottoCenter/grottocenter-api/blob/develop/api/services/GeoLocService.js#L4))

After adding quotes in the SQL, the entrance object will look like this: 
```
{
    id: 21,
    name: 'Saints de Glace (Les)',
    city: 'Autrans-Méaudre-en-Vercors',
    region: 'Isère (38), Auvergne-Rhône-Alpes (ARA)',
    cave: { id: 75075, name: null, depth: 670, length: 49300 },
    longitude: 5.520071983337401,
    latitude: 45.152929703286,
    quality: 10
}
```

Here the key path to depth, length where correct no need to change them.

Reply:
@vmarseguerra : I forgot to create the PR for the backend part. Please see it here : https://github.com/GrottoCenter/grottocenter-api/pull/1137

With this change in the backend API it should make more sense.